src/test/java/mate/academy/dao/impl/MovieDaoImplTest.java

```diff
@@ -0,0 +1,78 @@
+package mate.academy.dao.impl;
+
+import java.util.Optional;
+import mate.academy.dao.MovieDao;
+import mate.academy.model.Movie;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class MovieDaoImplTest extends AbstractTest {
+
+    @Override
+    protected Class<?>[] entities() {
+        return new Class[]{Movie.class};
+    }
+
+
+    @Test
+    public void create_Ok() {
+        MovieDao movieDao = new MovieDaoImpl(getSessionFactory());
+        insertMovies(movieDao);
+    }
+
+    @Test
+    public void getById_Ok() {
+        MovieDao movieDao = new MovieDaoImpl(getSessionFactory());
+        insertMovies(movieDao);
+
+        Optional<Movie> actualOptional = movieDao.get(1L);
+        Assert.assertTrue(actualOptional.isPresent());
+        Movie actual = actualOptional.get();
+        Assert.assertNotNull(actual);
+        Assert.assertEquals(1L, actual.getId().longValue());
+        Assert.assertEquals(Movies.shawshankRedemption.getTitle(), actual.getTitle());
+    }
+
+    @Test
+    public void getByNotExistingId_Ok() {
+        MovieDao movieDao = new MovieDaoImpl(getSessionFactory());
+        Optional<Movie> actual = movieDao.get(100L);
+        Assert.assertFalse(actual.isPresent());
+    }
+
+    private void insertMovies(MovieDao movieDao) {
+        Movie shawshankRedemption = Movies.shawshankRedemption.clone();
+        verifyCreateMovieWorks(movieDao, shawshankRedemption, 1L);
+
+        Movie godfather = Movies.godfather.clone();
```

Comment:
do we need to clone? probably we could create new objects and put movie names to constants if we don't want to hardcode them